viewing_party/party.py

```diff
@@ -1,23 +1,205 @@
 # ------------- WAVE 1 --------------------
+from tests.test_constants import *
+def create_movie(movie_title, genre, rating):
+    if movie_title == None or genre == None or rating == None:
+        return None
+    else:
+        new_movie = {"title":MOVIE_TITLE_1,"genre":GENRE_1,"rating":RATING_1}
+        return new_movie
 
-def create_movie(title, genre, rating):
-    pass
+def add_to_watched(user_data, movie):
+    user_data = {"watched":[{"title":MOVIE_TITLE_1, "genre":GENRE_1, "rating":RATING_1}]}

```

Comment:
This function will always return the same result.  The intention is to update the user_data argument that is being passed in and add the movie argument to the "watched" list of that user_data.
```suggestion
    user_data["watched"].append(movie)

```